### PR TITLE
Logaxis pan close to 0 fix

### PR DESCRIPTION
**Pull request by ralucaportase**

### CODE REVIEW

File: jquery.flot.navigate.js [hunk 1]
```diff
@@ -176,7 +175,6 @@ can set the default in the options.
             plot.getPlaceholder().css('cursor', plot.getOptions().pan.cursor);
             startPageX = e.pageX;
             startPageY = e.pageY;
-            plotState = plot.navigationState();
```
Comment (cipix2000) [inline jquery.flot.navigate.js]: this is important for smart pan as it does all the computation relative to the initial state

File: jquery.flot.logaxis.js [hunk 2]
```diff
@@ -74,6 +74,8 @@ Set axis.mode to "log" to enable.
 
             if (max < min) {
                 max = axis.max = axis.datamax | axis.options.max;
```
Comment (cristiingineru) [inline jquery.flot.logaxis.js]: The | operator is working on bits. Is this really working as supposed to?

Is it possible to write a test for this case? I'm sure someone will end up breaking something here again sooner or later.

File: jquery.flot.logaxis.js [hunk 3]
```diff
@@ -211,6 +217,16 @@ Set axis.mode to "log" to enable.
         return base * Math.floor(n / base);
     }
 
+    function setLogxisDataminRange(plot, axis) {
```
Comment (cristiingineru) [inline jquery.flot.logaxis.js]: Something is wrong about this name :)

File: jquery.flot.logaxis.js (hunk 2 above)
Comment (ralucaportase) [inline jquery.flot.logaxis.js]: I think the test that I wrote for this PL in logaxis.Test covers this case

File: tests/jquery.flot.logaxis.Test.js [hunk 4]
```diff
@@ -141,6 +141,22 @@ describe("unit tests for the log scale functions", function() {
...
+                    offset : { below: -1, above: -1 },
+                    max: 1
+                }]
+            }),
+            axis, ticks,
+
+        axis = plot.getAxes().xaxis;
+
+        expect(axis.min).toBe(0.1);
+        expect(axis.max).toBe(1);
```
Comment (cristiingineru) [inline tests/jquery.flot.logaxis.Test.js]: I think there should be 2 tests: one for expect(axis.min).toBeSmallerThan(axis.max), and a different one for the "0.1 constant" -> this is a small feature and it deserves its own test.

**Pull request merged**